src/Entity.cpp

```diff
@@ -11,6 +11,14 @@ m_size( j["width"].get<int>(), j["height"].get<int>() ),
 m_position( j["px"][0].get<int>(), j["px"][1].get<int>() ),
 m_grid_pos( j["__grid"][0].get<int>(), j["__grid"][1].get<int>() )
 {
+    if (j["__tile"] != nullptr)
```

Comment:
To remain coherent, I used `is_null` in other classes
```suggestion
    if (!j["__tile"].is_null())
```